devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -0,0 +1,162 @@
+/**
+ *  Copyright 2019 SRPOL
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition (name: "Z-Wave Basic Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-stateless-curtain") {
+		capability "Stateless Curtain Power Button"
+		capability "Configuration"
+		capability "Actuator"
+		capability "Health Check"
+
+		command "open"
+		command "close"
+		command "pause"
+
+		fingerprint mfr:"0086", prod:"0003", model:"008D", deviceJoinName: "Aeotec Nano Shutter"
+		fingerprint mfr:"0086", prod:"0103", model:"008D", deviceJoinName: "Aeotec Nano Shutter"
+	}
+
+	tiles(scale: 2) {
+ 		standardTile("open", "device.open", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Open', action:"open"
+		}
+		standardTile("close", "device.close", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Close', action:"close"
+		}
+		standardTile("pause", "device.pause", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label:'Pause', action:"pause"
+		}
+
+		details(["open","close","pause"])
+	}
+
+	preferences {
+		input (
+				title: "Working direction",
+				description: "In case wiring is wrong, this setting can be changed to fix setup without any manual maintenance.",
```

Comment:
Would it make our lives easier if this was a boolean that asked whether or not you want to reverse working direction?